CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## v4.5.0 - 2021-08-02
+
+**Important notes**
```

Comment:
Suggested test:

**Important notes**

A **BREAKING CHANGE** is introduced for JWT middleware users.
The JWT library used for the JWT middleware had to be changed from [github.com/dgrijalva/jwt-go](https://github.com/dgrijalva/jwt-go) to 
[github.com/golang-jwt/jwt](https://github.com/golang-jwt/jwt) due former library being unmaintained and affected by security 
issues. 
The `golang-jwt/jwt` project is a drop-in replacement, but supports only the lastest 2 Go versions.
So for JWT middleware users Go 1.15+ is required. For detailed information please read https://github.com/labstack/echo/discussions/1940

To change the library imports in all .go files in your project replace all occurences of `dgrijalva/jwt-go` with `golang-jwt/jwt`.

For Linux you can use:

```bash
find -type f -name "*.go" -exec sed -i "s/dgrijalva\/jwt-go/golang-jwt\/jwt/g" {} \;
go mod tidy
```

Reply:
I'm not sure the example to replace the library using `sed` is required. That's way I added the hint above as it will work for all users with their favorite plattform/IDE

Reply:
just to be inclusive even for those hardcore vim/nano users :)